lua/refactoring/region.lua

```diff
@@ -127,7 +127,7 @@ function Region:to_ts()
     -- Need the -2 for end_col to be  correct for `ts_utils.is_in_node_range`
     -- function results when checking scope for languages like python
     return self.start_row - 1,
-        self.start_col,
+        self.start_col - 1,
```

Comment:
While this clearly fixes a whole bunch of stuff, in the case where we are attempting to just extract one character in CPP,  I _believe_ this is causing a query error. However, on `master` right now, trying this case just results in it extracting nothing, which is not much better. For now I think this is better but this will definitely need more work in the future to perfect for different situations.

Reply:
@pranavrao145 I might throw in a ticket to add more tests around this to more accurately reflect it's behavior, maybe even as a `good_first_issue` task. But it works with existing tests,  so I think I'm good to merge.